apps/partners/src/utils/chainLogs.ts

```diff
@@ -0,0 +1,172 @@
+const mintTopic =
```

Comment:
Will this be different for mainnet? Do we need to put this in chain config or something?

Reply:
Same for mainnet and testnet, but since it's used internally in these methods, I don't think it would be necessary to move them